dl1_data_handler/reader.py

```diff
@@ -1180,6 +1220,14 @@ def __destructor(self):
     def _append_features(self, batch) -> Table:
         pass
 
+    @abstractmethod
+    def _get_raw_example(self, batch) -> Table:
+        pass
+
+    @abstractmethod
+    def _add_tdscan_table(self,batch) -> Table:
+        pass
```

Comment:
Why are they `abstractmethod` if they are only used in the trigger subclass? I think they should be removed here and elsewhere (except in the trigger subclass).